common/achievement/util.ts

```diff
@@ -33,55 +33,57 @@ export function getRelationTypeAndId(relation: string): [type: RelationTypes, id
     return [relationType as RelationTypes, id];
 }
 
-export async function getBucketContext(relation: string, id: string): Promise<AchievementContextType> {
-    const [userType, userId] = getUserTypeAndIdForUserId(id);
-    const [relationType, relationId] = getRelationTypeAndId(relation);
+// TODO: fix naming
+export async function getBucketContext(myUserID: string, relation?: string): Promise<AchievementContextType> {
+    const [userType, userId] = getUserTypeAndIdForUserId(myUserID);
```

Comment:
userId and myUserId are confusing.

Reply:
yeah, this is why there is an open todo 🙄 
I forgot to rename it...